src/app/(auth)/_components/email-send-form.tsx

```
@@ -0,0 +1,112 @@
+                                    </FormControl>
+                                    <FormMessage />
+                                </FormItem>
+                            )}
+                        />
+                    </div>
+                    <Button
+                        type="submit"
+                        // disabled={loading}
+                        className="w-full cursor-pointer"
+                    >
+                        {false ? <Spinner size="sm" /> : "Submit"}
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)